setup.py

```diff
@@ -28,15 +28,14 @@
 kerberos_require = ["requests_kerberos"]
 sqlalchemy_require = ["sqlalchemy~=1.3"]
 
-all_require = kerberos_require + sqlalchemy_require
+all_require = ["pytz"] + kerberos_require + sqlalchemy_require
```

Comment:
Move above as `timezone_require = ["pytz"]` and add here `timezone_require` to be consistent.

Reply:
kerberos and sqlalchemy are optional dependencies. I would say we need then another list called default_require, any other default dependency would then be added to that list.

Reply:
Let's keep as it is. The idea with the lists was to use them for optional deps.

If we find this list becoming large we can extract a separate list. No need to do it now.